### ISSUE TITLE
Adding compatibility with Windows NPS as the RADIUS server.

### ISSUE THREAD

**Issue by wkneeppel** [FreeRADIUS/mod_auth_radius#49]
TLDR: Apache calls this module, sends the request to NPS, NPS evaluates and returns an accept, but Apache does not process the accept.

Context:
Apache Subversion (CollabNet SVN Edge or VisualSVN Server) over HTTPS.
Running in Windows domain environment with external NAS/AD Servers.
Incorporating this module into the Apache server to use NPS as the RADIUS server.

Started with changes from Unicode to Winsock as suggested here: https://www.apachelounge.com/viewtopic.php?p=41452
Apache now sends requests.
NPS requires the use of Message-Authenticator attribute in RADIUS communications. Modified the module to include/calculate this attribute. (See files)
After properly configuring NPS it now processes the requests and sends back a radius-accept message.
Apache/mod_auth_radius is not handling the accept. Apache logs (svn access) shows no user in the SVN log.
To me it appears that the module is either not handling the response or it is not sending information back to Apache httpd in a way that is expected for apache basic authentication to complete.

[Mod_Auth_Radius for Windows.zip](https://github.com/user-attachments/files/16676052/Mod_Auth_Radius.for.Windows.zip)

**Comment by alandekok** [comment]
Please send this as a git pull request, with multiple commits.  It's extremely difficult to merge in changes in this format.

i.e. you've made a large number of changes, with no explanation.  

What did you change?  Why was it changed?
